web-components/src/[sandbox]/sandbox.ts

```diff
@@ -276,9 +284,7 @@ export class Sandbox extends LitElement {
 
           <div class="container" aria-label="md-form">
             <h2>md-form</h2>
-            <sass-stats component="form">
```

Comment:
why did you remove sass stats here?

Reply:
there are no styles for form, it's all derived from <input>, and therefore no stats get compiled for display.